contrib/checkbox-provider-ce-oem/units/iio-sensors/test-plan.pxu

```diff
@@ -0,0 +1,40 @@
+id: ce-oem-iio-sensors-full
+unit: test plan
+_name: Industrial I/O sensor tests
+_description: Full tests for Industrial I/O sensors
+include:
+nested_part:
+    ce-oem-iio-sensors-manual
+    ce-oem-iio-sensors-automated
+    after-suspend-ce-oem-iio-sensors-manual
+    after-suspend-ce-oem-iio-sensors-automated
+
+id: ce-oem-iio-sensors-manual
+unit: test plan
+_name: Industrial I/O sensor manual tests
+_description: Manual tests for Industrial I/O sensors
+include:
+
+id: ce-oem-iio-sensors-automated
+unit: test plan
+_name: Industrial I/O sensor auto tests
+_description: Automated tests for Industrial I/O sensors
+bootstrap_include:
+    ce-oem-iio-sensors/resource
+include:
+    ce-oem-iio-sensors/check-.*-.*
```

Comment:
This regular expression is almost the same as "ce-oem-iio-sensors/check-.*". What's the reason for using this "-.*-.*"?

Reply:
Fixed this issue by include the template-id in the test plan